frontend/packages/operator-lifecycle-manager/src/components/operand/create-operand.tsx

```diff
@@ -45,31 +43,27 @@ import { OperandYAML } from './operand-yaml';
 
 import './create-operand.scss';
 
-export const CreateOperand: React.FC<CreateOperandProps> = ({
-  clusterServiceVersion,
-  customResourceDefinition,
-  initialEditorType,
-  loaded,
-  loadError,
-  match,
-  model,
-}) => {
+export const CreateOperand: React.FC<CreateOperandProps> = ({ csv, initialEditorType, match }) => {
   const { t } = useTranslation();
+  const [model] = useK8sModel(match.params.plural);
+  const [crd] = useK8sWatchResource<CustomResourceDefinitionKind>({
+    kind: CustomResourceDefinitionModel.kind,
+    isList: false,
+    name: nameForModel(model),
+  });
```

Comment:
Need to make sure model is defined before we use it to fetch CRD:
```suggestion
  const [crd] = useK8sWatchResource<CustomResourceDefinitionKind>(
    model
      ? {
          kind: CustomResourceDefinitionModel.kind,
          isList: false,
          name: nameForModel(model),
        }
      : null,
  );
```